plugins/dell-dock/fu-dell-dock-i2c-ec.c

```diff
@@ -373,6 +373,7 @@ fu_dell_dock_ec_get_dock_info (FuDevice *device,
 			    device_entry[i].version.version_8[3]);
 			g_debug ("\tParsed version %s", self->ec_version);
 			fu_device_set_version (FU_DEVICE (self), self->ec_version);
+			fu_device_set_version_format (FU_DEVICE (self), FWUPD_VERSION_FORMAT_QUAD);
```

Comment:
I would think it makes sense to set this _before_ the call to `set_version` to avoid calling the guess logic every time.

Reply:
This one is less applicable since I realized you removed the guess.

Reply:
I'll provide a helper, it's too easy to get them in the wrong order, especially if we validate in the future.